controllers/bounce_processes.go

```diff
@@ -92,28 +88,18 @@ func (bounceProcesses) reconcile(ctx context.Context, r *FoundationDBClusterReco
 
 		addresses = append(addresses, addressMap[process]...)
 
-		processGroupID := podmanager.GetProcessGroupIDFromProcessID(process)
-		pod, err := r.PodLifecycleManager.GetPods(ctx, r, cluster, internal.GetSinglePodListOptions(cluster, processGroupID)...)
-		if err != nil {
-			return &requeue{curError: err}
-		}
-		if len(pod) == 0 {
```

Comment:
In order to catch this case we could check for `MissingPod` or `PendingPod` on the other hand a missing/pending Pod should not block any restarts, we can assume that the newly created Pod will get the new configuration directly.